tests/test_particle_track.py

```diff
@@ -179,69 +211,69 @@ class TestValueErrors:
     """
 
     def test_seedx(self):
-        params = particle_track.params()
+        params = particle_track.modelParams()
         # expect ValueError
         with pytest.raises(ValueError):
-            particle = particle_track.Particle(params)
+            particle = particle_track.Particles(params)
 
     def test_seedy(self):
-        params = particle_track.params()
+        params = particle_track.modelParams()
         params.seed_xloc = [1]
         # expect ValueError
         with pytest.raises(ValueError):
-            particle = particle_track.Particle(params)
+            particle = particle_track.Particles(params)
 
     def test_seedx_none(self):
         class pobj():
             pass
         params = pobj
         with pytest.raises(ValueError):
-            particle = particle_track.Particle(params)
+            particle = particle_track.Particles(params)
 
     def test_seedy_none(self):
         class pobj():
             pass
         params = pobj
         params.seed_xloc = [1]
         with pytest.raises(ValueError):
-            particle = particle_track.Particle(params)
+            particle = particle_track.Particles(params)
 
     def test_num_tracers(self):
-        params = particle_track.params()
+        params = particle_track.modelParams()
         params.seed_xloc = [1]
         params.seed_yloc = [1]
         with pytest.raises(ValueError):
-            particle = particle_track.Particle(params)
+            particle = particle_track.Particles(params)
```

Comment:
Particle attributes have moved out of `modelParams`, so these tests shouldn't expect a `ValueError`

Reply:
Can also delete the particle attributes from the rest of the `params` object tests

Reply:
Good points, thanks